src/pages/expenses/common/hooks.tsx

```diff
@@ -65,6 +74,47 @@ export const expenseColumns = [
 
 type ExpenseColumns = typeof expenseColumns[number];
 
+export function useActions() {
+  const [t] = useTranslation();
+
+  const navigate = useNavigate();
+
+  const [, setExpense] = useAtom(expenseAtom);
+
+  const [, setRecurringExpense] = useAtom(recurringExpenseAtom);
```

Comment:
To skip the getter for the state, you can use `useUpdateAtom`.

```ts
const setExpense = useUpdateAtom(expenseAtom);
```

https://jotai.org/docs/utils/use-update-atom

Reply:
replaced